ttnn/cpp/ttnn/operations/data_movement/permute/permute.cpp

```diff
@@ -257,7 +224,7 @@ ttnn::Tensor ExecutePermute::invoke(
     tt::stl::Span<const int64_t> dims,
     const std::optional<MemoryConfig>& memory_config,
     const std::optional<float>& pad_value) {
-    return invoke(DefaultQueueId, input_tensor, dims, memory_config, true, pad_value);
+    return invoke(DefaultQueueId, input_tensor, dims, memory_config, pad_value);
```

Comment:
nice, I have no idea what `true` meant before 👍

Reply:
I didn't know what it was used for but I avoided touching it to scope creep my PRs up until now.